xxh3.h

```diff
@@ -579,7 +579,7 @@ XXH3_len_9to16_64b(const xxh_u8* input, size_t len, const xxh_u8* secret, XXH64_
     XXH_ASSERT(input != NULL);
     XXH_ASSERT(secret != NULL);
     XXH_ASSERT(9 <= len && len <= 16);
-    {   xxh_u64 const input_lo = XXH_readLE64(input)           ^ (XXH_readLE64(secret)     + seed);
+    {   xxh_u64 const input_lo = XXH_readLE64(input)           ^  XXH_readLE64(secret);
```

Comment:
I presume this is to prevent the seed from being roughly cancelled out when we add them together below?

Reply:
Yes